src/modm/architecture/interface/can_message.hpp

```diff
@@ -88,6 +89,26 @@ struct Message
 		length = len;
 	}
 
+	inline uint8_t
+	getDLC() const
+	{
+		if(length <= 8) {
+			return length;
+		}
+		else if(length <= 24) {
+			return 8 + ((length - 8) / 4);
+		}
+		else {
+			return 12 + ((length - 24) / 8);
+		}
+	}
+
+	inline void
+	setDLC(uint8_t dlc)
+	{
+		length = dlcLookup.at(dlc);
```

Comment:
> If pos is not within the range of the container, an exception of type std::out_of_range is thrown.

Which is unhandled in modm and will do something weird like call std::terminate.
https://en.cppreference.com/w/cpp/container/array/at

Reply:
Yes, that should be changed if we want to merge it without CANFD. Didn't look at that so far since I didn't get to the long frames yet.

Reply:
I would suggest to keep `setDlc()` even without long-frame support at the moment, so that the interface is already there.

```suggestion
		if(dlc > dlcLookup.size())
			length = 0;
		else
			length = dlcLookup[dlc];
```

Reply:
Isn't this a source for buffer overflows? Most code will probably assume the message length fits into the data buffer. Let's put the decoding into some free function for now and clamp the length at 8 for the standard message type. When long frames are added, the lookup function is already there and can be reused. Ok?

Reply:
OK 👍🏽

Reply:
We decided to remove the feature for now. There were some design questions we couldn't find a spontaneous answer to.